### DISCUSSION TITLE
Not work

### DISCUSSION THREAD

**Discussion by amcucine** [community/community#118404] [Programming Help]
### Select Topic Area

Bug

### Body

Il mio sito l'ho messo online ma non applica le modiche che ho fatto con html css

**Comment by ebndev** [comment]
Hi [amcucine](https://github.com/amcucine), thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

You may want to post in our Programming Help 🧑‍💻 category next time.
